src/data/api.js

```diff
@@ -18,7 +23,7 @@ export async function wrappedFetch(url, body) {
 
   // reminder: fetch does not throw exceptions for non-200 responses (https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch)
   const response = await fetch(`${process.env.BASE_API_URL}/${url}`, {
```

Comment:
Do we want to put this in a try catch statement in case of network errors?

Reply:
Probably so. I was about to pull in the entire `customFetch` function from TF but decided to stay focused on the REST migration. This improvement should probably come very soon.